caddyfile.go

```diff
@@ -35,7 +35,7 @@ func parseApp(d *caddyfile.Dispenser, _ interface{}) (interface{}, error) {
 	for d.NextBlock(0) {
 		switch d.Val() {
 		case "domains":
-			for d.NextBlock(0) {
+			for d.NextBlock(1) {
```

Comment:
Wonder if we should do the other form mentioned in the godoc for NextBlock. That way we don't have to hope it's 1.

Reply:
You mean `nesting := d.Nesting();`? Sure